### DISCUSSION TITLE
How to use google Auth on localhost(in development) and website domain(production) at the same time?

### DISCUSSION THREAD

**Discussion by roker15** [supabase/supabase#3703] [Questions] [answered]
I am trying to login on localhost using google sign in provided by supabase... but it is getting me directed to my website after login. If i change setting of **Site URL** in **authenticaiton setting on supabase** to localhost then my website is not able to login customer. In nutshell i am able to use google auth either at localhost or at my domain.. i want to use both

**Comment by silentworks** [comment, marked as answer]
Follow what I have answered here https://github.com/supabase/supabase/discussions/2760#discussioncomment-1144158 and make sure to setup the **Additional Redirect URLs** to have the other domain in it. The url you are using in the `redirectTo` must be exactly the same as the one in **Additional Redirect URLs** in order for it to work.
